en/news/_posts/2017-12-25-ruby-2-5-0-released.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: news_post
+title: "Ruby 2.5.0 Released"
+author: "naruse"
+translator:
+date: 2017-12-25 00:00:00 +0000
+lang: en
+---
+
+We are pleased to announce the release of Ruby 2.5.0.
+
+Ruby 2.5.0 is the first stable release of Ruby 2.5 series.
+It introduces many new features and performance improvements, for example:
+
+## New Features
+
+* Print backtrace and error message in reverse order if STDERR is unchanged and a tty.
+  [&#x5B;Feature #8661&#x5D;](https://bugs.ruby-lang.org/issues/8661) [experimental]
+
+* Top-level constant look-up is removed.  [&#x5B;Feature #11547&#x5D;](https://bugs.ruby-lang.org/issues/11547)
+
+* rescue/else/ensure are allowed inside do/end blocks.  [&#x5B;Feature #12906&#x5D;](https://bugs.ruby-lang.org/issues/12906)
```

Comment:
`allows for using rescue/else/ensure keywords directly with do/end blocks`

Reply:
„inside“ seems clearer